evalml/automl/automl_search.py

```diff
@@ -318,6 +321,10 @@ def __init__(self,
                                        num_ensemble_batches)
             else:
                 self.max_iterations = 1 + len(self.allowed_pipelines) + (self._pipelines_per_batch * (self.max_batches - 1))
+        if run_ensembling:
+            self.X_train, self.X_ensemble, self.y_train, self.y_ensemble = split_data(self.X_train, self.y_train, problem_type=self.problem_type, test_size=0.20)
```

Comment:
@chukarsten right, I see your point. Through my discussions with @dsherry, we decided that a 0.8/0.2 split seemed fine for separating the ensembling, but I do see your concern about making it more adjustable/customizable. I don't see a problem with adding it as a (private for now) arg to AutoMLSearch, in case a user wants to make it finer? 

What do you think, @dsherry?

Reply:
If you guys chatted about it, let's just file an issue to handle it later, then!

Reply:
I'll leave this up for discussion for now, in case other people have inputs.

Reply:
Ah yes, let's please make this configurable! I think its fine to default it to 20%.